setup.py

```diff
@@ -7,13 +7,24 @@
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
+
+def find_scripts():
+    scripts = []
+    exclude = ['setup.py']
```

Comment:
Should we also exclude common.py, as this isn't intended to be callable and seems likely to have naming collisions?

Reply:
Shouldn't be required as `scandir` doesn't recurse over the folder structure and only searches for `.py` files in the root folder.

Reply:
@crahan That was my oversight, apologies. I hadn't considered the code in setup.py in the context of the post-merge directory structure.